core/templates/pages/teach-page/teach-page.controller.ts

```diff
@@ -77,17 +69,17 @@ angular.module('oppia').directive('teachPage', [
             var hash = WindowRef.nativeWindow.location.hash.slice(1);
 
             if (hash === ctrl.TAB_ID_TEACH) {
-              activateTab(ctrl.TAB_ID_TEACH);
+              ctrl.activeTabName = ctrl.TAB_ID_TEACH;
             } else if (hash === ctrl.TAB_ID_PLAYBOOK) {
-              activateTab(ctrl.TAB_ID_PLAYBOOK);
+              ctrl.activeTabName = ctrl.TAB_ID_PLAYBOOK;
             }
```

Comment:
Would just `ctrl.activeTabName = hash;` suffice here? Or if you want to check that the hash is in the `[ctrl.TAB_ID_TEACH, ctrl.TAB_ID_PLAYBOOK]` you can probably make a helper function and use it also on lines 79 to 83.

Reply:
Done.

Reply:
Can we get rid of the separate ifs then and do just `ctrl.activeTabName = hash`?

Reply:
@marianazangrossi What about this?